Minor doc fixes (#1671)
zeit/docs: pages/docs/v2/platform/deployments.mdx

The **preview URL** is provided on a pull or merge request when using a [ZEIT Now for Git Integration](/docs/v2/git-integrations) and contains the name of the user or team to which the [Project](/docs/v2/platform/projects) belongs.

> If you request a **preview URL**, the `X-Robots-Tag` HTTP header will be set as `noindex`. That means preview URLs will NOT be indexed by search engines like Google. Learn more about `X-Robots-Tag` [here](https://developers.google.com/search/reference/robots_meta_tag#xrobotstag).

Reviewer: ```suggestion
If you request a **preview URL**, the `X-Robots-Tag` HTTP header will be set as `noindex`. This ensures that preview URLs will _not_ be indexed by search engines such as Google. Learn more about `X-Robots-Tag` by visiting [Google Search Reference](https://developers.google.com/search/reference/robots_meta_tag#xrobotstag).
```
